bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -202,6 +205,8 @@ protected void initModel() {
                 baseResourcePath = resource.getPath();
             }
             baseResourcePath = resource.getResourceResolver().map(request, baseResourcePath);
+            //get asset title or name from dam
```

Comment:
I think we can remove this comment altogether now that we don't look up the title. The function call below is itself readable.